app/services/search_indexer.rb

```diff
@@ -31,7 +31,7 @@ def self.update_index(table, id, raw_data)
     end
 
     # for user login and name use "simple" lowercase stemmer
-    stemmer = table == "user" ? "simple" : Search.long_locale
+    stemmer = table == "user" ? "simple" : Search.ts_config
```

Comment:
hmm why was this change to use ts_config? I guess long_local may not even be available in pg?